src/line-numbers-tile-component.coffee

```diff
@@ -106,9 +113,11 @@ class LineNumbersTileComponent
     else
       lineNumber = (bufferRow + 1).toString()
 
-    padding = _.multiplyString('&nbsp;', maxLineNumberDigits - lineNumber.length)
-    iconHTML = '<div class="icon-right"></div>'
-    padding + lineNumber + iconHTML
+    padding = _.multiplyString("\u00a0", maxLineNumberDigits - lineNumber.length)
+    iconRight = @elementsPool.build("div", "icon-right")
+
+    lineNumberNode.innerText = padding + lineNumber
```

Comment:
I guess I got confused with `innerHTML`: apparently `textContent` is faster and, moreover, is consistent with our codebase.

So, :+1: Thank you